sqlalchemy_batch_inserts/__init__.py

```diff
@@ -32,16 +32,23 @@ def _has_normal_id_primary_key(base_mapper):
         python_column_type = None
 
     return (
-        primary_key_col.name == "id"
-        and python_column_type == int
-        and primary_key_col.autoincrement in ("auto", True)
-        and primary_key_col.table == base_mapper.local_table
+            python_column_type == int
+            and primary_key_col.autoincrement in ("auto", True)
+            and primary_key_col.table == base_mapper.local_table
     )
 
 
+def _get_primary_key_name(base_mapper):
+    primary_key_cols = base_mapper.primary_key
+    [primary_key_col] = primary_key_cols
+    return primary_key_col.name
+
+
 def _get_id_sequence_name(base_mapper):
-    assert _has_normal_id_primary_key(base_mapper), "_get_id_sequence_name only supports id primary keys"
-    return "%s_id_seq" % base_mapper.entity.__tablename__
+    assert _has_normal_id_primary_key(base_mapper), \
+        "_get_id_sequence_name only supports single auto increment primary keys"
+    primary_key = _get_primary_key_name(base_mapper)
+    return f"{base_mapper.entity.__tablename__}_{primary_key}_seq"
```

Comment:
This brought up a good discussion on how backward compatible we should be with Python 2. For this release, our answer is yes, let's remain backward compatible.

Accordingly, could you change to the old-style string formatting from before?

```suggestion
"%s_%s_seq" % (base_mapper.entity.__tablename__, primary_key)
```